### PR TITLE
[Merged by Bors] - chore: remove unused `cases'` names

### PR DESCRIPTION
**Pull request by kim-em**
These unused names will soon acquire a linter warning. Easiest to clean them up pre-emptively.

(Thanks to @nomeata for fixing these on `nightly-testing`.)

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Positivity/Core.lean [hunk 1]
```diff
@@ -74,7 +74,7 @@ initialize positivityExt : PersistentEnvExtension Entry (Entry × PositivityExt)
     (List Entry × DiscrTree PositivityExt) ←
   -- we only need this to deduplicate entries in the DiscrTree
   have : BEq PositivityExt := ⟨fun _ _ => false⟩
-  let insert kss v dt := kss.foldl (fun dt ks => dt.insertCore ks v discrTreeConfig) dt
```
Comment (eric-wieser) [inline Mathlib/Tactic/Positivity/Core.lean:77]: This one looks stray

**Review by eric-wieser — APPROVED** [review]
bors d+

File: Mathlib/Tactic/Positivity/Core.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Tactic/Positivity/Core.lean]: Thanks, reverted.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**